debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+                esriStyle["Golf course"                  ] = vectorTileStyling.landuse;
+                esriStyle["Industry/label"               ] = [];
+                esriStyle["Marine area/label"            ] = [];
+                esriStyle["Railroad"                     ] = vectorTileStyling.road;
+                esriStyle["Medical"                      ] = vectorTileStyling.landuse;
+                esriStyle["Education"                    ] = vectorTileStyling.landuse;
+                esriStyle["Park or farming"              ] = vectorTileStyling.landuse;
+                esriStyle["Exit"                         ] = vectorTileStyling.road;
+                esriStyle["Retail/label"                 ] = [];
+                esriStyle["Beach/label"                  ] = [];
+                esriStyle["Beach"                        ] = vectorTileStyling.landuse;
+                esriStyle["Special area of interest"     ] = vectorTileStyling.housenumber;
+                esriStyle["Point of interest"            ] = vectorTileStyling.poi;
+                esriStyle["Education/label"              ] = [];
+                esriStyle["Landmark"                     ] = vectorTileStyling.landuse;
+                esriStyle["Cemetery"                     ] = vectorTileStyling.landuse;
+                esriStyle["Transportation"               ] = vectorTileStyling.road;
+                esriStyle["Landmark/label"               ] = [];
+                esriStyle["Medical/label"                ] = [];
+                esriStyle["Park or farming/label"        ] = [];
+                esriStyle["Building/label"               ] = [];
+
+                const esriTilesUrl = "https://basemaps.arcgis.com/v1/arcgis/rest/services/World_Basemap/VectorTileServer/tile/{z}/{y}/{x}.pbf";
+
+                const esriVectorTileOptions = {
+                    rendererFactory: L.canvas.tile,
+                    attribution: '© ESRI',
+                    vectorTileLayerStyles: esriStyle,
+                };
+
+                let esriTilesPbfLayer = L.vectorGrid.protobuf(esriTilesUrl, esriVectorTileOptions);
+
+				// Sliced GeoJSON code copied and updated from:
+				// https://leaflet.github.io/Leaflet.VectorGrid/demo-geojson.html
+
+				let euJsonLayer = L.vectorGrid.slicer( euCountries, {
+				rendererFactory: L.svg.tile,
+				vectorTileLayerStyles: {
+					sliced: function(properties, zoom) {
+						let p = properties.mapcolor7 % 5;
+						return {
+							fillColor: p === 0 ? '#800026' :
+								p === 1 ? '#E31A1C' :
+								p === 2 ? '#FEB24C' :
+								p === 3 ? '#B2FE4C' : '#FFEDA0',
+							fillOpacity: 0.5,
+							stroke: true,
+							fill: true,
+							color: 'black',
+							weight: 0,
+						}
+					}
+				},
+				getFeatureId: function(f) {
+					return f.properties.wb_a3;
+				}
+            }
+		);
+
+		// Set an extent that includes BC as well as Europe
+		leafletMap.setView({ lat: 48, lng: -40 }, 3);
```

Comment:
Consider including the initial extent in the smk-config.json instead. It looks a little strange for the map to start at the extent of BC and then zoom out.

Reply:
Done.